src/assets/SSSAsset.js

```diff
@@ -0,0 +1,35 @@
+const postcss = require('postcss');
+const sugarss = require('sugarss');
+const CSSAsset = require('./CSSAsset.js');
+
+class SSSAsset extends CSSAsset {
+  constructor(name, options) {
+    options.css_parser = "sugarss";
```

Comment:
Why is this necessary?

Reply:
I wanted to propagate to `transform` to automatically load parser, if it is necessary.

But not sure if it is good idea

Reply:
You already do a localRequire in transform so not sure why this is necessary at all.